src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+body,
+a,
+ul,
+li {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
do not hardcode min and max width, it should be responsive

Reply:
deleted